src/main/java/com/zaxxer/hikari/util/PropertyElf.java

```diff
@@ -186,6 +194,54 @@ private static String capitalizedPropertyName(String propertyName)
       return propertyName.substring(0, 1).toUpperCase(Locale.ENGLISH) + propertyName.substring(1);
    }
 
+   private static int[] parseIntArray(String value)
+   {
+      if( value == null || value.isEmpty() ) {
```

Comment:
Project style is space after ``if``, no space after ``(``. For example:
```
if (value == null || value.isEmpty()) {
```

Same change everywhere pls.